client/src/contexts/index.jsx

```diff
@@ -44,18 +41,6 @@ export function AuthProvider({ children }) {
     return signOut(auth);
   }
 
-  function resetPassword(email) {
```

Comment:
Did you decide against incorporating reset password, update email/password for this first iteration of the app?

Reply:
Yeah - I figure it will keep things cleaner in the context/index.js file